tests/h/search/core_test.py

```diff
@@ -16,6 +16,7 @@
 from h import search
 
 
+@pytest.mark.usefixtures("group_service", "moderation_service")
```

Comment:
The only two tests that need these (technically they just need the group_service) in this class are the separate_wildcard_uri_keys tests that don't depend on the Annotation fixture. It feels a little weird to me that such a simple test would need so much. I wonder if we should just mock all the query module in this case. These two tests only need the group_service because it is used by the group filter which is really independent of these tests.